source/PlayerInfoPanel.cpp

```diff
@@ -559,35 +641,62 @@ void PlayerInfoPanel::DrawFleet(const Rectangle &bounds)
 	Color bright = *GameData::Colors().Get("bright");
 	Color elsewhere = *GameData::Colors().Get("dim");
 	Color dead(.4f, 0.f, 0.f, 0.f);
+	Color special(0.5f, 0.5f, 0.1f, 0.f);
+
+	// Table columns and their starting x positions, alignment and sort comparator
+	columns = {
```

Comment:
Should we be reassigning this every time `DrawFleet` is called?

Reply:
I guess not, I assumed there was a reason why everything was being reassigned but I can move them out